translations/CONTRIBUTING.french.md

```diff
@@ -25,25 +25,48 @@ combinaison du nom de la branche et du SHA de commit git.
 
 Pour toutes les contributions, un CLC (Contat de Licence de Contributeur) doit être signé [ici](https://cla-assistant.io/hasura/graphql-engine) avant (ou après) que la pull request ait été soumise. Un bot demandera si nécessaire aux contributeurs de signer le CLC au moyen d'un commentaire de pull request.
 
+## Bienvenue aux contributeurs débutants!
+
+Nous accueillons les nouveaux contributeurs avec joie et serions heureux de vous aider dans vos débuts! Si vous avez des questions, n'hésitez pas!
+
+Vous trouverez toutes les issues adaptées aux débutants [ici](https://github.com/hasura/graphql-engine/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
+
+## Domaines de contribution
+
+Nous apprécions bien sûr toute contribution pour Hasura, peu importe le composant. Cependant, nous avons déterminé trois domaines particulièrement adaptés à des contributions en open source.
 
 ### Documentation
 
+Notre objectif est d'avoir une documentation intuitive et à jour. Si vous souhaitez nous y aider, nous acceptons toute contribution avec joie:
+
+- Signaler du contenu manquant
+
+- Régler les erreurs sur les docs existants
+
+- Ajouter du contenu à la documentation 
+
 Le guide de contribution pour la documentation peut être trouvé sur [docs/CONTRIBUTING.md](docs/CONTRIBUTING.md).
```

Comment:
I've just noticed that this link is broken. Could you update it please? 🙂

Reply:
Oops sorry, I didn't see that...
It should work properly now